forest/cli/Cargo.toml

```diff
@@ -0,0 +1,92 @@
+[package]
+name        = "forest-cli"
+version     = "0.4.0"
+authors     = ["ChainSafe Systems <info@chainsafe.io>"]
+description = "Filecoin implementation in Rust. This crate contains all the subcommands except the daemon."
+edition     = "2021"
+
+[dependencies]
+anyhow.workspace                 = true
+async-log                        = "2.0"
+async-std                        = { workspace = true, features = ["attributes", "tokio1"] }
+base64.workspace                 = true
+byte-unit                        = "4.0"
+chrono                           = "0.4"
+cid                              = { workspace = true, default-features = false, features = ["std"] }
+directories                      = "4.0.1"
+fil_actor_miner_v8.workspace     = true
+forest_actor_interface.workspace = true
+forest_auth.workspace            = true
+forest_beacon.workspace          = true
+forest_blocks.workspace          = true
+forest_chain.workspace           = true
+forest_chain_sync.workspace      = true
+forest_cli_shared.path           = "../shared"
```

Comment:
Should we you write it here or would it be better to move it to top-level `Cargo.toml` as in e.g.lines 112-149

Reply:
fixed